rust/hacl-sys/build.rs

```diff
@@ -25,11 +25,9 @@ fn create_bindings(include_path: &Path, home_dir: &Path) {
         .allowlist_function("EverCrypt_HMAC_.*")
         .allowlist_function("Hacl_P256_.*")
         .allowlist_function("Hacl_RSAPSS_.*")
-        .allowlist_function("Hacl_SHA3_.*")
-        .allowlist_function("Hacl_Chacha20Poly1305_.*")
+        .allowlist_function("Hacl_AEAD_Chacha20Poly1305_.*")
         .allowlist_function("Hacl_Hash_.*")
         .allowlist_function("Hacl_Streaming_.*")
```

Comment:
Does this still exist?

Reply:
I meant the `Hacl_Streaming`. I thought that's all renamed?

Reply:
Indeed!